readme.md

```diff
@@ -30,6 +38,7 @@ A couple of default styles lifted from [Marked.app][marked].
 
 [mmdql]: https://github.com/fletcher/MMD-QuickLook
 [screenshot]: https://github.com/ttscoff/MMD-QuickLook/raw/master/MMDQLGithub-1.jpg
+[cask]: http://caskroom.io/
```

Comment:
Ah, yes, you are correct. I was thinking `cask` would be the name to appear, but `homebrew-cask` is. My mistake.